pynetbox/__init__.py

```diff
@@ -1,6 +1,7 @@
 from pkg_resources import get_distribution, DistributionNotFound
 
-from pynetbox.core.query import RequestError
+from pynetbox.core.query import RequestError, AllocationError
+from pynetbox.core.query import ContentError
```

Comment:
I think this would be ok on L3 or did the linter complain?

Reply:
Oh, yes, no complaints now that I joined the lines. I just thought the limit was 72 characters! :-D

Reply:
Nice. Thanks!